CONTRIBUTING.md

```diff
@@ -22,10 +26,74 @@ TODO
 * CI
 
 ### Python Development Guidelines
+Related Python projects are grouped in repos, each repo containing a number of application projects: `cadctools`, `caom2tools`, `vostools`. A project of general interest might be published on `PyPI` (The Python Project Index).
+The OpenCADC Python software is expected to be available and work with all the official active versions of Python (see https://devguide.python.org/versions/.)
+
+#### Directory Structure
+A project, let's say `myapp` has the following structure:
+`myapp` root directory with the following files:
+* `setup.py` - script used by `pip` command to run install the application
+* `setup.cfg` - configuration file for `pip`. It contains the dependencies and the version of the application.
+* `README.md` - description of the project
+* `LICENCE` - the license file
+* `tox.ini` - `tox` configuration file
```

Comment:
None of the `*2caom2` applications have a `tox.ini` configuration in place. Maybe this is better phrased as "required for projects of general interest, that are to be published to `PyPI`?

I could add `tox` directives to all the `*2caom2` apps, but as I'm usually the only developer, that is work that will not help anyone. Let me know what you think I should do.